test/unit_tests/team/TeamPermissionSpec.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {ROLE, roleFromPermissions} from 'src/script/team/TeamPermission';
+
+const memberPermissionBitmask = 1587;
```

Comment:
Can we please write those in binary notation, so they make more sense as a bitmask?
```suggestion
const memberPermissionBitmask = 0b11000110011;
```